### DISCUSSION TITLE
List of grammar errors to be fixed within Mastodon [ITALIAN]

### DISCUSSION THREAD

**Discussion by ihealyou** [mastodon/mastodon#20080] [General]
I found no other place to report this so here we go. I'll keep editing this thread if I notice any other grammar mistakes. Feel free to add others if you found any. [ITALIAN ONLY]

![1](https://user-images.githubusercontent.com/117687782/200430790-3ceaa493-6fef-4a6b-a65f-3ff19b8ff29e.PNG)
(should be "Argomento"/"Argomenti")

**Comment by Lastorder-DC** [comment]
https://crowdin.com/project/mastodon

You can contribute/discuss about translation at crowdin.

**Reply by ihealyou** [reply]
Thanks!
